tests/reftests/env.test

```diff
@@ -439,60 +439,6 @@ The following actions will be performed:
 Done.
 ### opam exec -- sh -c "eval $(opam env | tr -d '\\r'); opam remove foo; opam env; eval $(opam env | tr -d '\\r'); opam env" | grep "FOO"
 FOO=''; export FOO;
-### : root and switch with spaces :
-### RT="$BASEDIR/root 2"
-### SW="switch w spaces"
-### OPAMNOENVNOTICE=0
-### opam init -na --bare --bypass-check --disable-sandbox --root "$RT" defaut ./REPO | grep -v Cygwin
-No configuration file found, using built-in defaults.
-
-<><> Fetching repository information ><><><><><><><><><><><><><><><><><><><><><>
-[defaut] Initialised
-### opam switch create "./$SW" nv --root "$RT"
-
-<><> Installing new switch packages <><><><><><><><><><><><><><><><><><><><><><>
-Switch invariant: ["nv"]
-
-<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
--> installed nv.1
-Done.
-# Run eval $(opam env '--root=${BASEDIR}/root 2' '--switch=${BASEDIR}/switch w spaces') to update the current shell environment
-### opam env --root "$RT" --switch "./$SW" | grep "NV_VARS" | ';' -> ':'
```

Comment:
why is this test not needed anymore? What was it testing to being with?

Reply:
I'm not seeing that command in either of those files

Reply:
ah,the command itself, true.

Reply:
I think that originally it was here to show that the switch was well created?

Reply:
updated

Reply:
This is safe - the main thing is that the primary tests in #5935 (further up the file) are still there. The noise in these other parts suggests that the tests should possibly be split up a bit further (but's _definitely_ a post 2.2.0 thing!)